openedx/core/lib/tests/test_edx_api_utils.py

```diff
@@ -195,3 +196,11 @@ def test_data_retrieval_failure(self, mock_exception):
 
         self.assertTrue(mock_exception.called)
         self.assertEqual(actual, [])
+
+    @httpretty.activate
+    @mock.patch('edx_rest_api_client.client.EdxRestApiClient.__init__')
+    def test_client_passed(self, mock_init):
+        """Verify that when API client is passed edx_rest_api_client is not used"""
+        program_config = self.create_programs_config()
+        get_edx_api_data(program_config, self.user, 'orders', Client())
```

Comment:
To be complete, it would make sense to use the Slumber client returned by `ecommerce_api_client`.